src/DynamoCoreWpf/ViewModels/Search/BrowserItemViewModel.cs

```diff
@@ -89,7 +89,7 @@ public interface ISearchEntryViewModel : INotifyPropertyChanged, IDisposable
         ElementTypes ElementType { get; }
     }
 
-    public class NodeCategoryViewModel : NotificationObject, ISearchEntryViewModel
+    public class NodeCategoryViewModel : ViewModelBase, ISearchEntryViewModel 
```

Comment:
same here. can this be accomplished without changing the base class by just adding more interfaces?

Reply:
you can check this doc: 
https://stackoverflow.com/questions/1456785/a-definitive-guide-to-api-breaking-changes-in-net

and do some thought experiments or real experiments with extensions using that class before and after.

Reply:
`public abstract class ViewModelBase : NotificationObject,IDisposable` I dont think switching out the baseclass here break API backward compatibility because it only adds two APIs
```
IsDebugBuild
```
and
```
Dispose
```